.github/ISSUE_TEMPLATE/000-incubation-review.yaml

```diff
@@ -0,0 +1,73 @@
+  on an early-stage design direction.
+title: "Incubation: "
+labels:
+  - "Progress: untriaged"
+  - "Review type: CG early review"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for sending us your design review.
+
+        Use links to content rather than pasting text into this issue. Issues are ephemeral and most of the material we are asking for has long term value.
+
+        Please preview the issue and check that the links work before submitting. Please make sure anyone with the link can access the document. We may refuse to review anything that is not public.
+
+  - type: input
+    attributes:
+      label: Explainer
+      description: An explainer must address user needs and contain examples of use. See our [explanation of how to write a good explainer](https://w3ctag.github.io/explainer-explainer/#introduction).
+      placeholder: Enter a URL.
+      value: "https://"
+    validations:
+      required: true
+
+  - type: checkboxes
+    attributes:
+      label: The explainer
+      options:
+        - label: Includes the information requested by the [Explainer Explainer](https://w3ctag.github.io/explainer-explainer/#introduction).
+          required: true
+        - label: Follows the [Web Platform Design Principles](https://www.w3.org/TR/design-principles/).
+        - label: Includes or links to answers to the [Security/Privacy Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/).
+        - label: Describes user research you did to validate the problem and/or design.
+
+  - type: textarea
+    attributes:
+      label: Where and by whom is the work is being done?
+      value: |
+        - GitHub repo:
+        - Primary contacts:
+          - $name (@-mention), $organization/s, $role in developing feature
+          - <!-- repeat as necessary, we recommend including group chairs and editors in this list -->
+        - Organization/project driving the design:
+        - This work is being funded by:
+        - Incubation and standards groups that have discussed the design:
+          - {{ABC CG}} <!-- Include a link to minutes or issues in this group if possible. -->
+          - {{DEF WG}}
+        - Standards group(s) that you expect to discuss and/or adopt this work when it's
+          ready: <!-- "unknown" if not known -->
+
+  - type: textarea
+    attributes:
+      label: Feedback so far
+      description: |
+        For your own organization, you can simply state the organization's position instead of linking to it.  This includes items on [Mozilla standards-positions](https://github.com/mozilla/standards-positions), and [WebKit standards-positions](https://github.com/WebKit/standards-positions).  Chromium doesn't have a standards-positions repository and [prefers](https://source.chromium.org/chromium/chromium/src/+/main:docs/standards/positions/GoogleChrome/README.md) to use comments from the teams that maintain the relevant area of their codebase.
```

Comment:
```suggestion
        For your own organization, you can simply state the organization's position instead of linking to it.  This includes items on [Mozilla standards-positions](https://github.com/mozilla/standards-positions), and [WebKit standards-positions](https://github.com/WebKit/standards-positions). Chromium doesn't have a standards-positions repository and [prefers](https://source.chromium.org/chromium/chromium/src/+/main:docs/standards/positions/GoogleChrome/README.md) to use comments from the teams that maintain the relevant area of their codebase.
```